zip-0000.rst

```diff
@@ -0,0 +1,583 @@
+should abide by the Zcash Code of Conduct [#conduct]_
+and follow `the GNU Kind Communication
+Guidelines <https://www.gnu.org/philosophy/kind-communication.en.html>`__
+
+
+ZIP format and structure
+========================
+
+ZIPs SHOULD be written either in reStructuredText [#rst]_ or LaTeX [#latex]_.
+In the latter case, a `Makefile` MUST be provided to build (at least) a
+PDF version of the document.
+
+Each ZIP SHOULD have the following parts:
+
+* Preamble -- Headers containing metadata about the ZIP (`see
+  below <#zip-header-preamble>`__).
+  The License field of the preamble indicates the licensing terms,
+  which MUST be acceptable according to `the ZIP licensing requirements <#zip-licensing>`__.
+
+* Terminology -- Definitions of technical or non-obvious terms used
+  in the document.
+
+* Abstract -- A short (~200 word) description of the technical issue
+  being addressed.
+
+* Motivation -- The motivation is critical for ZIPs that want to change
+  the Zcash protocol. It should clearly explain why the existing
+  protocol is inadequate to address the problem that the ZIP solves.
+
+* Specification -- The technical specification should describe the
+  interface and semantics of any new feature. The specification should be
+  detailed enough to allow competing, interoperable implementations for
+  any of the current Zcash platforms.
+
+* Rationale -- The rationale fleshes out the specification by
+  describing what motivated the design and why particular design
+  decisions were made. It should describe alternate designs that were
+  considered and related work. The rationale should provide evidence of
+  consensus within the community and discuss important objections or
+  concerns raised during discussion.
+
+* Security and privacy considerations -- If applicable, security
+  and privacy considerations should be explicitly described, particularly
+  if the ZIP makes explicit trade-offs or assumptions. For guidance on
+  this section consider `RFC 3552 <https://tools.ietf.org/html/rfc3552>`__.
+  as a starting point.
+
+* Reference implementation -- Literal code implementing the ZIP's
+  specification, and/or a link to the reference implementation of
+  the ZIP's specification. The reference implementation must be
+  completed before any ZIP is given status “Implemented”, but it
+  generally need not be completed before the ZIP is accepted into
+  Proposed.
+
+ZIP header preamble
+-------------------
+
+Each ZIP must begin with an RFC 822-style header preamble. The following
+header fields are REQUIRED::
+
+  ZIP:
+  Title:
+  Owners:
+  Status:
+  Category:
+  Created:
+  License:
+
+The following additional header fields are OPTIONAL::
+
+  Discussions-To:
+  Network Upgrade:
+  Obsoleted by:
+  Updated by:
+  Obsoletes:
+  Updates:
+
+The Owners header lists the names and email addresses of all the
+champions/owners of the ZIP. The format of the Owners header value
```

Comment:
Is "champions" purposely left here?

Reply:
Thanks for the review @rex4539 — in my search/replace I neglected to consider case-insensitivity, just pushed a commit to fix :raised_hands: